docs/ui-toolkit/components/_posts/1970-01-13-Image.md

```diff
@@ -15,7 +15,7 @@ This document covers `Image` component and Image size style names available in S
 
 #### Props
 
-* `Image` component doesn't have specific (custom) Props, however, it supports every prop that the standard React Native `Image` component supports. For full list of available props, visit
+* `Image` doesn't have specific (custom) Props, however, it supports every prop that the standard React Native `Image` component supports. For full list of available props, visit
```

Comment:
`has no`, `(component) doesn't have` the same thing is written in a few styles. We should choose one.